README.md

```diff
@@ -1,9 +1,19 @@
 [![][kuma-logo]][kuma-url]
 
-[![CircleCI](https://circleci.com/gh/kumahq/kuma/tree/release-1.4.svg?style=svg&circle-token=e3f6c5429ee47ca0eb4bd2542e4b8801a7856373)](https://circleci.com/gh/kumahq/kuma/tree/release-1.4)
+**Builds**
+- Release-1.5 [![CircleCI release-1.5](https://circleci.com/gh/kumahq/kuma/tree/release-1.5.svg?style=shield)](https://circleci.com/gh/kumahq/kuma/tree/release-1.5)
+- Master [![CircleCI master](https://circleci.com/gh/kumahq/kuma/tree/master.svg?style=shield)](https://circleci.com/gh/kumahq/kuma/tree/master)
```

Comment:
very picky perhaps but I think you can add `&label=release` and `&label=master` to have `[release | passing]` and `[master | passing]` and skip the bullet points

Reply:
Ho not at all! I looked for this and couldn't find it thanks!